django/db/models/fields/related.py

```diff
@@ -975,6 +975,7 @@ def formfield(self, *, using=None, **kwargs):
             raise ValueError("Cannot create form field for %r yet, because "
                              "its related model %r has not been loaded yet" %
                              (self.name, self.remote_field.model))
+        kwargs.update({'blank': self.blank})
```

Comment:
You can just add `'blank': self.blank,` underneath `**kwargs,` below.

Reply:
Thanks @pope1ni